src/clr-core/styles/layout/docs/layout.stories.mdx

```diff
@@ -85,6 +85,25 @@ the component. [Read More: Grid Layout](?path=/docs/experimental-layout-grid--pa
 </div>
 ```
 
+## Responsive
+
+The layout system provides many built-in responsive utilities to make it easy to
+manage layouts on many different screen sizes. All responsive utilities follow the same set of breakpoints.
+
+<Story id="experimental-design-tokens-stories--layout" />
+
+```css
+@media (min-width: 480px) { }
```

Comment:
We may need to consider aligning these with the existing clarity breakpoints. In apps that use both Core and Clarity Angular/UI, responsive behavior would look weird as Core components would shift at different screen sizes from the other components.

I have no problem changing breakpoints globally but I think this is a bigger change that will have a hard time flying under the radar.

Reply:
I can update them in this PR. I'll also update the example to pull from the token so they are not hard coded.